src/Renderer/PointsMaterial.js

```diff
@@ -82,29 +247,38 @@ class PointsMaterial extends THREE.RawShaderMaterial {
      * pointMaterial.recomputeClassification();
      */
     constructor(options = {}) {
-        const intensityRange = options.intensityRange || new THREE.Vector2(0, 1);
+        const intensityRange = options.intensityRange || new THREE.Vector2(1, 65536);
+        const elevationRange = options.elevationRange || new THREE.Vector2(0, 1000);
+        const angleRange = options.angleRange || new THREE.Vector2(-90, 90);
         const oiMaterial = options.orientedImageMaterial;
-        const classification = options.classification || ClassificationScheme.DEFAULT;
+        const classifScheme = options.classification || classificationLUT.DEFAULT;
+        const otherScheme = options.other || otherLUT.DEFAULT;
         const applyOpacityClassication = options.applyOpacityClassication == undefined ? false : options.applyOpacityClassication;
         const size = options.size || 0;
         const mode = options.mode || PNTS_MODE.COLOR;
         const shape = options.shape || PNTS_SHAPE.CIRCLE;
         const sizeMode = size === 0 ? PNTS_SIZE_MODE.ATTENUATED : (options.sizeMode || PNTS_SIZE_MODE.VALUE);
         const minAttenuatedSize = options.minAttenuatedSize || 3;
         const maxAttenuatedSize = options.maxAttenuatedSize || 10;
+        const _gradient = options.gradient || 'SPECTRAL';
 
-        delete options.orientedImageMaterial;
         delete options.intensityRange;
-        delete options.classification;
+        delete options.elevationRange;
+        delete options.angleRange;
+        delete options.orientedImageMaterial;
+        delete options.classifScheme;
+        delete options.otherScheme;
         delete options.applyOpacityClassication;
         delete options.size;
         delete options.mode;
         delete options.shape;
         delete options.sizeMode;
         delete options.minAttenuatedSize;
         delete options.maxAttenuatedSize;
+        delete options.gradient;
 
         super(options);
+        this.gradient = _gradient;
```

Comment:
So `gradient` attribute is only an identifier? What if a user wants to provide its own gradient?

Reply:
Done,
Users can now add their own gradient (should be in the same format as the one in Utils/Gradients.js) with options.gradient.